azure-pipelines.release.yml

```diff
@@ -0,0 +1,76 @@
+pr: none
+trigger: none
+
+variables:
+  - group: "Github and NPM secrets"
+
+pool:
+  vmImage: "Ubuntu 16.04"
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: "8.x"
+    displayName: "Install Node.js"
+
+  - script: |
+      git config user.name "UI Fabric Build"
+      git config user.email "fabrictactical@service.microsoft.com"
+      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/OfficeDev/office-ui-fabric-react.git
+    displayName: Authenticate git for pushes
+
+  - script: |
+      node common/scripts/install-run-rush.js install --bypass-policy
+    displayName: rush install
+
+  - script: |
+      npm run build
+    displayName: rush build (Create production build)
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: apps/fabric-website/dist
+      artifactName: "fabric-website"
+      publishLocation: "Container"
+    displayName: "Publish Artifact: Fabric Website"
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: apps/fabric-website/index.html
+      artifactName: "fabric-website-index"
+      publishLocation: "Container"
+    displayName: "Publish Artifact: Fabric Website index.html"
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: packages/office-ui-fabric-react/dist
+      artifactName: "fabric"
+      publishLocation: "Container"
+    displayName: "Publish Artifact: Fabric"
+
+  - script: |
+      node ./common/scripts/install-run-rush.js publish -a -p -b 5.0 -t lts -n $(npmToken) --add-commit-details --set-access-level public
```

Comment:
we currently have 6.0 as "lts" as well... @jdhuntington which should it be?

Reply:
Oh good catch, I didn't see that flag...

Reply:
Probably the real question is, who's using the `lts` tag and why? Guessing it should continue pointing to 5 and **not** 6.

Reply:
Likely folks who come across the project via NPM

![image](https://user-images.githubusercontent.com/706967/59544922-e3a11200-8ecb-11e9-9dc6-c3863c9b8320.png)

Reply:
Removed the lts tag from this release pipeline